react-app/src/api/submissionService.ts

```diff
@@ -63,8 +63,7 @@ export const buildSubmissionPayload = <T extends Record<string, unknown>>(
 ) => {
   const userDetails = {
     submitterEmail: user?.user?.email ?? "N/A",
-    submitterName:
-      `${user?.user?.given_name} ${user?.user?.family_name}` ?? "N/A",
```

Comment:
The fallback would never run since the first value is a string. I'm gonna change the check so it falls back if the user object is empty

Reply:
True. Yeah, because nullish operator only checks for null or undefined. Wondering if we should flip to a || instead or just remove?

Reply:
Resolved with https://github.com/Enterprise-CMCS/macpro-mako/pull/704/commits/fe8de8b3dd6dfa0329d0635a52970754e04fd5d2